internal/service/evidently/wait.go

```diff
@@ -0,0 +1,59 @@
+package evidently
+
+import (
+	"time"
+
+	"github.com/aws/aws-sdk-go/service/cloudwatchevidently"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func waitProjectCreated(conn *cloudwatchevidently.CloudWatchEvidently, nameOrARN string, timeout time.Duration) (*cloudwatchevidently.GetProjectOutput, error) {
```

Comment:
`statusProject` calls `FindProjectByNameOrARN` which returns `cloudwatchevidently.Project`. We should return the same type as the finder

```suggestion
func waitProjectCreated(conn *cloudwatchevidently.CloudWatchEvidently, nameOrARN string, timeout time.Duration) (*cloudwatchevidently.Project, error) {
```